theme/static/style/style.css

```diff
@@ -98,6 +98,50 @@ ul.topnav li [name="q"] {
     border: 0;
 }
 
+.topnav{
+    overflow: hidden;
+    box-shadow: 0 0.2rem 0.5rem 0 rgba(0,0,0, .3);
+    z-index: 1;
+}
+
+.topnav ul{
+    list-style-type: none;
+    text-align: right;
+}
+
+.topnav li{
+    display: inline-block;
+    vertical-align: top;
+}
+
+.topnav ul a{
+    display: inline-block;
+    padding: 0 1.6rem;
+    text-align: center;
+    line-height: 5rem;
+    border-radius: 0.3rem;
+}
+
+.topnav .search{
```

Comment:
isn't this selector a duplicate for `ul.topnav li [name="q"]` which already exists? If necessary update definitions in existing declaration.

Reply:
also delete this block